Tests/tests_e2e/content/xsoar_saas/test_e2e_xsoar_saas.py

```diff
@@ -54,21 +54,28 @@ def test_edl_returns_indicators(request: SubRequest, xsoar_saas_client: XsoarSaa
         username = integration_params["credentials"]["identifier"]
         password = integration_params["credentials"]["password"]
 
-        with save_integration_instance(
-            xsoar_saas_client,
-            integration_params=integration_params,
-            integration_id="EDL",
-            is_long_running=True,
-            instance_name=get_integration_instance_name(integration_params, default="EDL")
-        ) as edl_instance_response:
-            instance_name = edl_instance_response.get("name")
-
-            edl_response = xsoar_saas_client.do_long_running_instance_request(
-                instance_name, username=username, password=password
-            )
-            assert edl_response.text, f'could not get indicators from url={edl_response.request.url} from ' \
-                f'instance {instance_name} with available indicators={available_indicators},' \
-                f' status code={edl_response.status_code}, response={edl_response.text}'
+        for i in range(1, 6):
+            with save_integration_instance(
+                xsoar_saas_client,
+                integration_params=integration_params,
+                integration_id="EDL",
+                is_long_running=True,
+                instance_name=get_integration_instance_name(integration_params, default="EDL")
+            ) as edl_instance_response:
+                instance_name = edl_instance_response.get("name")
+                try:
+                    edl_response = xsoar_saas_client.do_long_running_instance_request(
+                        instance_name, username=username, password=password
+                    )
+                    assert edl_response.status_code == 200, f"status code is {edl_response.status_code} " \
+                        f"from EDL in URL {edl_response.request.url}"
+                    text = edl_response.text
+                    assert "1.1.1.1" in text, f"1.1.1.1 indicator from URL {edl_response.request.url} is not in {text}"
+                    assert "2.2.2.2" in text, f"2.2.2.2 indicator from URL {edl_response.request.url} is not in {text}"
+                    assert "3.3.3.3" in text, f"3.3.3.3 indicator from URL {edl_response.request.url} is not in {text}"
+                    break
+                except Exception as error:
+                    logging.error(f"test_edl_returns_indicators - time {i} - error\n{error}")
```

Comment:
instead of time  i would use try / try #

Reply:
i like `time` more, makes more sense IMO